dstep/core/Traits.d

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright: Copyright (c) 2017 Jacob Carlborg. All rights reserved.
+ * License: $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0)
+ */
+module dstep.core.Traits;
+
+bool isKeyword (string ident)
+{
+    import std.algorithm.searching;
+
+    static immutable DKeywords = [
+        "abstract", "alias", "align", "asm", "assert", "auto", "body", "bool",
+        "break", "byte", "case", "cast", "catch", "cdouble", "cent", "cfloat",
+        "char", "class", "const", "continue", "creal", "dchar", "debug",
+        "default", "delegate", "delete", "deprecated", "do", "double", "else",
+        "enum", "export", "extern", "false", "final", "finally", "float", "for",
+        "foreach", "foreach_reverse", "function", "goto", "idouble", "if",
+        "ifloat", "immutable", "import", "in", "inout", "int", "interface",
+        "invariant", "ireal", "is", "lazy", "long", "macro", "mixin", "module",
+        "new", "nothrow", "null", "out", "override", "package", "pragma",
+        "private", "protected", "public", "pure", "real", "ref", "return",
+        "scope", "shared", "short", "static", "struct", "super", "switch",
+        "synchronized", "template", "this", "throw", "true", "try", "typedef",
+        "typeid", "typeof", "ubyte", "ucent", "uint", "ulong", "union",
+        "unittest", "ushort", "version", "void", "volatile", "wchar", "while",
+        "with"
+    ];
+ 
+    return DKeywords.canFind(ident);
+}
```

Comment:
There is already an existing function for this: https://github.com/jacob-carlborg/dstep/blob/master/dstep/translator/Translator.d#L487